ui2/src/api/api.js

```diff
@@ -37,19 +37,59 @@ export const login = async user => {
   }
 };
 
+export const getAuthToken = () => localStorage.getItem(AUTH_TOKEN);
+
 export const appInit = () => {
-  const token = localStorage.getItem(AUTH_TOKEN);
-  let loggedIn = false;
+  const token = getAuthToken();
   if (token) {
     axios.defaults.headers.common['Authorization'] = token;
-    loggedIn = true;
   } else {
     logout();
   }
 
-  return loggedIn;
+  return !!token;
 };
 
 export const appLogout = () => {
   localStorage.removeItem(AUTH_TOKEN);
 };
+
+export const getServices = async () => {
+  try {
+    const { data = null } = await axios.get(`${baseUrl}/service/`);
+    return data;
+  } catch (error) {
+    console.error(error);
+    throw error;
+  }
+};
+
+export const getServicesStack = async () => {
+  try {
+    const { data = null } = await axios.get(`${baseUrl}/services-stack/`);
+    return data;
+  } catch (error) {
+    console.error(error);
+    throw error;
+  }
+};
+
+export const getEnvironment = async () => {
+  try {
+    const { data = null } = await axios.get(`${baseUrl}/environment/`);
+    return data;
+  } catch (error) {
+    console.error(error);
+    throw error;
+  }
+};
+
+export const getEnvironmentsStack = async () => {
+  try {
+    const { data = null } = await axios.get(`${baseUrl}/environments-stack/`);
+    return data;
+  } catch (error) {
+    console.error(error);
+    throw error;
+  }
```

Comment:
you can  warp 
const fetchData = async endPoint => {
  try {
    const { data = null } = await axios.get(`${baseUrl}/${endPoint}/`);
    return data;
  } catch (error) {
    console.error(error);
    throw error;
  }

  export const getEnvironmentsStack = async () =>  await fetchData('environments-stack')
  export const getEnvironment = async () =>  await fetchData('environment')
....

Reply:
done